rotkehlchen/chain/bitcoin/bch/utils.py

```diff
@@ -0,0 +1,199 @@
+from typing import List, Optional, Set, Tuple, Union
+
+import bech32
+from base58 import b58decode_check, b58encode_check
+from eth_typing import ChecksumAddress
+from marshmallow import ValidationError
+
+from rotkehlchen.chain.bitcoin.utils import is_valid_btc_address
+from rotkehlchen.types import BTCAddress
+
+_CHARSET = 'qpzry9x8gf2tvdw0s3jn54khce6mua7l'
+_VERSION_MAP = {
+    'legacy': [
+        ('P2SH', 5, False),
+        ('P2PKH', 0, False),
+    ],
+    'cash': [
+        ('P2SH', 8, False),
+        ('P2PKH', 0, False),
+    ],
+}
+_PREFIX = 'bitcoincash'
+
+
+def _polymod(values: list) -> int:
```

Comment:
Can you maybe add a docstring here, explaining what it is you are doing and also where this code was taken from? I guess it was taken from same place as the others? So perhaps that comment can go as docstring in the top of the module?